README.md

```diff
@@ -42,6 +45,11 @@ Or install it yourself as:
 
 ## Usage
 
+In order for Percona to work, your migrations have to be written using LHM's
+DSL. Although we plan to remove this dependency soo, it's still a requirement.
+You can find further details in its
+[repository](https://github.com/soundcloud/lhm/blob/master/README.md)
```

Comment:
It would be good to put an example of how a normal migration (like changing the table of a model) looks like.